content/docs/details/entity-push-api/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Entity Push API
+publishedDate: '2024-02-16'
+description: How to use the Entity Push API
+---
+
+# Catalog your AWS accounts
+
+In this tutorial we are going to show you how to ingest your organization's AWS accounts into your Roadie catalog as Resource entities. For this we will use Roadie's Entity Push API you can check out the api docs [here](https://roadie.io/docs/api/catalog/) in the Roadie Provider section
+
+## Authentication
+
+First to be able to use the roadie entity push api we will need to get an authentication token. You can generate your own token by going to the `Administration` -> `Account` (`/administration/account`) page. Here go to the `Roadie API Access` section. Give a name to your token and press the `GENERATE TOKEN` button.
+Make sure you copy your token and put it in a secure place.
+You can test out your token by hitting the api. For example:
+
+```bash
+curl \
+  -X GET \
+  -H 'Accept: application/json' \
+  -H "Authorization: bearer ${ROADIE_API_TOKEN}" \
+  https://api.roadie.so/api/catalog/entities
```

Comment:
Ive never been 100% sure on this endpoint as an example. It returns every entity in the catalog.. Maybe there is a better endpoing.. but not a big deal.

Reply:
will be fine imo